x-pack/legacy/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -130,6 +167,11 @@ export function dataFrameAnalyticsRoutes({ xpackMainPlugin, router }: RouteIniti
     })
   );
 
+  /**
+   * @api {post} /api/ml/data_frame/_evaluate Gets stats for requested analytics job
```

Comment:
Copy and paste error - this endpoint evaluates the data frame analytics for an annotated index.

Reply:
https://github.com/elastic/kibana/pull/56006/commits/fc9297f3498431f41f91440b5c9ffa664401a3c5